app/src/main/java/com/sopt/now/compose/ui/ch_password/ChPasswordScreen.kt

```diff
@@ -0,0 +1,160 @@
+        }
+    }
+
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        snackbarHost = { SnackbarHost(hostState = snackbarHostState) },
+        bottomBar = {}
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(innerPadding)
+                .padding(horizontal = 20.dp, vertical = 20.dp),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(
+                stringResource(R.string.ch_pw),
+                fontSize = 30.sp,
+            )
+
+            Spacer(Modifier.height(20.dp))
+            OutlinedTextField(
+                value = prePw,
+                onValueChange = { prePw = it },
+                label = { Text(stringResource(R.string.pre_pw)) },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                    keyboardType = KeyboardType.Password
+                ),
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier.fillMaxWidth()
+            )
+
+            Spacer(Modifier.height(16.dp))
+            OutlinedTextField(
+                value = newPw,
+                onValueChange = { newPw = it },
+                label = { Text(stringResource(R.string.new_pw)) },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                    keyboardType = KeyboardType.Password
+                ),
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier.fillMaxWidth()
+            )
+
+            Spacer(Modifier.height(16.dp))
+            OutlinedTextField(
+                value = verifyPw,
+                onValueChange = { verifyPw = it },
+                label = { Text(stringResource(R.string.verify_pw)) },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions(
+                    imeAction = ImeAction.Next,
+                    keyboardType = KeyboardType.Password
+                ),
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier.fillMaxWidth()
+            )
+
+            Spacer(modifier = Modifier.weight(1f))
+            CustomBtn(
+                text = stringResource(R.string.btn_ch_pw),
+                onClick = {
+                    viewModel.chPassword(RequestChPwDto(prePw, newPw, verifyPw))
+                },
+                modifier = Modifier.fillMaxWidth()
+            )
+        }
+    }
+}
+
+@Composable
+fun CustomBtn(
+    text: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        contentAlignment = Alignment.Center,
+        modifier = modifier
+            .fillMaxWidth()
+            .background(
+                MaterialTheme.colorScheme.primary,
```

Comment:
```suggestion
                color = MaterialTheme.colorScheme.primary,
```

named argument 빼먹으신 것 같아요